ImageItem/ImageItem.module.less

```diff
@@ -0,0 +1,28 @@
+// ImageItem.module.less
+//
+
+@import '../styles/skin.less';
+@import '../styles/mixins.less';
+
+.imageItem {
+    .applySkins({
+      .caption {
+        /* Public Class Names */
+        font-weight: @agate-imageitem-caption-weight;
```

Comment:
If the font-weight is same in all skins, we could define it out of the `applySkins`.

Reply:
If you want to support only silicon skin, please define @agate-imageitem-caption-weight in variables-base.less and variables-silicon.less. Then we could add @agate-imageitem-caption-weight in other variables later when needed.

Reply:
I removed it because is the same as the base font-weight